test/transformation.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from "chai";
+import { SearchHandler, defaultFileGroups, allLangs, allPlatforms } from "./testing-library";
+
+describe("Transformation Tests", () => {
+    it("should not have any files with leftover reserved words", async () => {
+        const result = await SearchHandler.searchInFiles(
+            [/\{Platform\}|\{ProductName\}/gm],
```

Comment:
you should get a list of variables from [docConfig.json](https://github.com/IgniteUI/igniteui-xplat-docs/blob/vnext/docConfig.json#L6) where we define them in 4 places (under each platform)